lib/alpaca_proxy/api.ex

```diff
@@ -28,21 +28,21 @@ defmodule AlpacaProxy.API do
     async_fetch!(
       build_url(config, conn),
       conn.method,
-      Map.to_list(conn.body_params),
+      conn.body_params,
       headers
     )
   end
 
   @spec async_fetch!(String.t(), method :: String.t(), body_params(), headers()) ::
           AsyncResponse.t()
-  defp async_fetch!(url, "GET", [], headers) do
+  defp async_fetch!(url, "GET", %{}, headers) do
```

Comment:
`%{}` does not match on `empty` map which is common mistake in `Elixir`:
```elixir
iex> match?(%{}, %{a: 5})
true
```
use [Kernel.map_size/1](https://hexdocs.pm/elixir/Kernel.html#map_size/1) instead:
```diff
- defp async_fetch!(url, "GET", %{}, headers) do
+ defp async_fetch!(url, "GET", map, headers) when map_size(map) == 0 do
```